ui-service/src/main/frontend/hotel-booking-ui/src/components/RecentSearch/types.ts

```diff
@@ -0,0 +1,31 @@
+export interface SearchFormData {
+  hotelType: "single" | "multi";
+  destination: string;
+  startDate: Date;
+  endDate: Date;
```

Comment:
Just FYI, in order to add multi hotel search, I updated @TengLuo's search form data to make this a list, { destination, startDate, endDate} []. We can talk about if that is reasonable.